projectroles/tasks.py

```diff
@@ -0,0 +1,47 @@
+"""Celery tasks for the samplesheets app"""
+
+import logging
+
+from django.conf import settings
+
+from config.celery import app
+
+# Projectroles dependency
+from projectroles.remote_projects import RemoteProjectAPI
+from projectroles.constants import SODAR_CONSTANTS
+from projectroles.models import RemoteSite
+
+
+remote_api = RemoteProjectAPI()
```

Comment:
From the error messages, it seems this init here in the module root creates the dependency problem that breaks CI. Maybe try moving the init inside `sync_remote_site_task()`?